sql/hive/src/main/scala/org/apache/spark/sql/hive/HiveContext.scala

```diff
@@ -615,11 +615,13 @@ class HiveContext private[hive](
         command.executeCollect().map(_.getString(0))
```

Comment:
don't we need to do it here as well? More generally should we wrap it around the whole `stringResult`?